docs/usage.md

```diff
@@ -1,4 +1,5 @@
 ---
+title: DietPi Usage Hints
```

Comment:
Not required, since the navigation menu uses "Usage hints / HowTo" already.

Reply:
Changed.